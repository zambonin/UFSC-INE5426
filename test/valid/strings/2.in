char let = 'a'
char wd[4], result[10]
wd = "luka"
# i can add letters together
result = wd + let
let = result[1]
