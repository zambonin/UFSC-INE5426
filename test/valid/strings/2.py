exec(open('../../../src/scope_manager.py', 'r').read())
let = 'a'
wd = [0] * 4
result = [0] * 10
wd = "luka"
result = (wd + str(let))
let = result[1]
