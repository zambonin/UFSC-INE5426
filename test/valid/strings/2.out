char var: let = 'a'
char array: wd (size: 4), result (size: 10)
= wd "luka"
= result + wd [word] let
= let [index] result 1
